__tests__/helpers/BasicPointsAndBadges/ConnectExternalAccountActivity.test.ts

```diff
@@ -0,0 +1,115 @@
+        "id" : 2,
+        "provider": { "id": 22, "name": "Fitbit", "category": "Device Manufacturer"},
+        status: "fetchComplete"
+    },
+    {
+        "id" : 3,
+        "provider": { "id": 22, "name": "FitBit", "category": "Device Manufacturer"} ,
+        status: "unauthorized"
+    },
+    {
+        "id" : 4,
+        "provider": { "id": 44, "name": "Health Plan 1", "category": "Health Plan"} ,
+        status: "fetchComplete"
+    }];
+
+    return {
+        __esModule: true,
+        default: {
+            token: { access_token: '1' },
+            getCurrentLanguage: () => jest.fn(),
+            getExternalAccounts: jest.fn(() => {
+                return Promise.resolve(externalAccounts);
+            }),
+            on: jest.fn(),
+            off: jest.fn()
+        }
+    }
+});
+
+describe("ConnectExternalAccountActivity Awards", () => {
+    it("should award points for connecting a single external account", async () => {
+         const activity = {
+             type: "connectExternalAccount",
+             points: 10,
+             providerCategories: ["Health Plan"]
+         } as ConnectExternalAccountActivity;
+         const activityState = {
+             pointsAwarded: 0,
+             providersConnected: []
+         };
+         const newActivityState = await awardConnectExternalAccountActivityPoints(activity, activityState);
+         expect(newActivityState.pointsAwarded).toBe(10);
+         expect(newActivityState.providersConnected).toEqual([44]);
+    });
+
+    it("should award points for connecting multiple external accounts", async () => {
+        const activity = {
+            type: "connectExternalAccount",
+            points: 10,
+            providerCategories: ["Health Plan", 'Provider', 'Device Manufacturer']
+        } as ConnectExternalAccountActivity;
+        const activityState = {
+            pointsAwarded: 0,
+            providersConnected: []
+        };
+        const newActivityState = await awardConnectExternalAccountActivityPoints(activity, activityState);
+        expect(newActivityState.pointsAwarded).toBe(30);
+        expect(newActivityState.providersConnected).toEqual([11, 22, 44]);
+   });
+
+    it("should NOT award points for multiple external accounts connected to same provider", async () => {
+        const activity = {
+            type: "connectExternalAccount",
+            points: 10,
+            providerCategories: ["Device Manufacturer"]
+        } as ConnectExternalAccountActivity;
+        const activityState = {
+            pointsAwarded: 0,
+            providersConnected: []
+        };
+        const newActivityState = await awardConnectExternalAccountActivityPoints(activity, activityState);
+        expect(newActivityState.pointsAwarded).toBe(20);
+        expect(newActivityState.providersConnected).toEqual([11, 22]);
+   });
+
+   it("should recaluclate points across all providers. Awarding points should be based on newly connected", async () => {
```

Comment:
Minor:  Typo in recalculate.  I think the prior description was fine too, since we aren't actually recomputing the value, we are just appending points for the newly connected ones, but I don't feel that strongly about it either way.